tools/legacy/BuildMonitor/Legacy_BuildMonitor.csproj

```diff
@@ -38,6 +38,7 @@
     <IncludeDebugSymbolsInLocalVSIXDeployment>true</IncludeDebugSymbolsInLocalVSIXDeployment>
     <CopyBuildOutputToOutputDirectory>true</CopyBuildOutputToOutputDirectory>
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+    <DeployExtension>False</DeployExtension>
```

Comment:
these were in the in original. They are coming back from reverting the revert. I was seeing some instability in tools.sln builds without this. @memontic-ms says I don't these now but they don't hurt anything to my knowledge so I left them.